### ISSUE TITLE
Raider is now opensource.

### ISSUE THREAD

**Issue by AssetKid** [AssetKid/raider-release#110]
TL:DR; Raider is now opensource. Feel free to fork.

---

Hey,

I have decided to release the Raider code to the public.

This project was forked from an old repo that was not maintained anymore. I cleaned up the code a bit and fixed most of the bugs but there was a lot of work required.

In December 2019, I started working on Raider v2.0 to address most of the issues in the current app. The UI was mostly done by early 2020.

![1](https://user-images.githubusercontent.com/40426812/171900753-30a12dfb-92f1-4d92-b144-b4ab2a95ed90.gif)
![2](https://user-images.githubusercontent.com/40426812/171902931-dc659af1-6a7f-4f8a-a298-3ed63d7c04c2.jpg)
![3](https://user-images.githubusercontent.com/40426812/171902939-4b9772d2-ceb1-40f1-9013-0397a669400e.jpg)
![4](https://user-images.githubusercontent.com/40426812/171902941-958f6ab0-8ff2-4a3f-b66b-3b462d762d83.jpg)

I work as a freelancer and once the pandemic hit, getting gigs became harder. The situation has not improved that much since then.

These types of projects (Freeware software) rarely bring in any income. Since July 2019, the Patreon has made $264.79 (After Patreon and billing charges).

![Patreon Stats](https://user-images.githubusercontent.com/40426812/171900714-afb1ee84-737d-4f6a-a930-d75de6bf9c32.jpeg)

Instagram makes quite a number of changes to the site and Raider has to be updated regularly to keep up with these changes. Before, this wasn't a big deal and I was more than happy to work on the app during my free time. 

This is no longer feasible.

I have received a few offers from companies that sell Instagram bot services to include ads, trackers etc. but I have turned them all down. As promised;

> Raider DOES NOT contain a single ad, any trackers or in app purchases. It is completely free to use.

Feel free to fork the repo. Be warned though, the code is a mess.

Apologize for any inconvenience this might cause.

For everyone that donated to the project, thank you very much.

Cheers.

**Comment by desbest** [comment]
One month has passed and there's since then been no new updates to Raider.
Is anyone going to maintain this?
If not, then it looks like I'll have to, as I can do programming.
You know what they say, Too many cooks spoil the broth.
So I'd rather not maintain Raider in future if someone else is going to.
If no one else is interested, then I'll start to maintain it.

**Comment by afterdelight** [comment]
pls fork it
